### DISCUSSION TITLE
Database Patterns: além do SQL na mão - Dev Hudde

### DISCUSSION THREAD

**Discussion by Junior-Shyko** [secultce/mapasculturais-v5#145] [Anúncios]
## ✅ Introdução
Veja a ideia da nossa Dev Hudde nesse link: https://github.com/secultce/mapasculturais-v5/discussions/132

## ✨ Tema 
**Database Patterns: além do SQL na mão**

## 🥳 Colaborador
@alessandrofeitoza 

## 🗓️ Dia do Evento
Sexta-feira, 5 de julho · 10:30 até 11:00 - ( 05/07/24 )

## 👥 Sala do Evento
Database Patterns: além do SQL na mão
Fuso horário: America/Fortaleza
Como participar do Google Meet
Link da videochamada: https://meet.google.com/dms-btzy-rsu

**Comment by TalysonSoares** [comment]
Muito bom! Estarei presente

**Comment by saracamilo** [comment]
Estarei acompanhando 👏

**Comment by henriquelopeslima** [comment]
Confirmando minha presença.

**Comment by ronnyjohnti** [comment]
Ótimo! Estarei presente

**Comment by jeff-doliveira1** [comment]
Estaremos lá
